open-metadata-implementation/adapters/open-connectors/connector-configuration-factory/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/ConnectorConfigurationFactory.java

```diff
@@ -699,9 +700,34 @@ public Connection getSecuritySyncServerConnection(String              serverName
         String connectionName = "SecuritySyncServer.Connection." + serverName;
         final String connectionGUID        = "GPKHhpfW-xXjN-QbFR-T50l-XMnY68SS2Bip";
         final String connectionDescription = "OMRS repository connection to Security Sync Server.";
-        return getConnection(configurationProperties, endpoint, connectionName, connectionGUID, connectionDescription);
+        return getConnection(configurationProperties, endpoint, connectionName,
+                connectionGUID, connectionDescription, RangerSecurityServiceConnectorProvider.class.getName());
     }
 
+    /**
+     * Return the connection.
+     *
+     * @param serverName  name of the real repository server
+     * @param url  url for the Security Officer Server
+     * @param configurationProperties name value pairs for the connection
+     * @return Connection object
+     */
+    public Connection getSecurityOfficerServerConnection(String              serverName,
+                                                         String              url,
+                                                         Map<String, Object> configurationProperties)
+    {
+        final String endpointGUID          = "P66yJ8gI-gJaZ-VOHw-b5fu-bfdilclSvfm1";
```

Comment:
GUIds are typically hex --this looks quite different to that which we usually use. Whilst it is interpreted as a string, I think we should stick with a standard approach?

Reply:
updated in the last commit.

Reply:
Thanks -- I saw most have been changed - just noticed one left :-)